tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,38 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario: upload a file with virus to a project space
+    Given using spaces DAV path
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "new-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "uploadFolder" in space "new-space"
+    When user "Alice" uploads a file inside space "new-space" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
```

Comment:
does this step check the existence of file in the project space you created or in personal space?

Reply:
true @nabim777  this checks in personal space we need to check in project space. We have steps like this we can use for that
```console
And for user "Alice" the space "new-space" should contain these entries:
      | /uploadFolder/virusFile.txt |
```